### PR TITLE
test fixes

### PR DESCRIPTION
**Pull request by tjake**
Needed for 9795

### CODE REVIEW

File: cqlsh_tests/cqlsh_copy_tests.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 import codecs
 import csv
 import datetime
+import time
 import locale
 import os
 import sys
+import platform
```
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_copy_tests.py]: You're not using either of these two new imports.

**Comment by ptnapoleon** [comment]
Also, travis is complaining about several whitespace violations, but those aren't supercritical. I can fix them after the merge if you don't have time

**Comment by tjake** [comment]
fixed

**Comment by ptnapoleon** [comment]
Merging, then fixing in new PR

**Pull request merged**